images/tidb-backup-manager/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM pingcap/tidb-enterprise-tools:latest
```

Comment:
There should be lightning in the base image, please confirm this.

Reply:
No, pingcap/tidb-enterprise-tools only have syncer and load.

Reply:
ok.